package/wpe/wpewebkit/Config.in

```diff
@@ -66,7 +66,7 @@ if BR2_PACKAGE_WPEWEBKIT
 
 choice
         bool "version"
-        default BR2_PACKAGE_WPEWEBKIT2_28
```

Comment:
Hi @modeveci please also check if https://github.com/WebPlatformForEmbedded/lithosphere needs to be updated as well for this. 
(I checked the gstreamer we use in lithosphere and it is already 1.22.8)

Reply:
Hi @MFransen69, I will update the default of wpewebkit for lithosphere as well. 
gstreamer from buildroot upstream is most probably the latest stable of the community. It should work with wpe-2.38, as the igalia team follows gstreamer activities too. 
I am merging this PR. Thanks